[DOCS] Add known issue with Observability Overview in 8.7.0 (#154593)
elastic/kibana: docs/CHANGELOG.asciidoc

[float]
[[known-issues-8.7.0]]
=== Known issues

// tag::known-issue-151698[]
[discrete]
.Observability Overview shows empty User Experience panel
[%collapsible]
====
*Details* +
Releases 8.7.0 has a bug which causes the Observability Overview page shows empty User Experience panel, even when there is RUM data (fixed in {kibana-pull}154419[#154419]).

*Impact* +

> Whilst the User Experience panel is empty on the Observability Overview page, if you have RUM data it will still be available from the User Experience Dashboard.

Reviewer: ```suggestion
While the User Experience panel on the Observability Overview page is empty, any RUM data will still be available from the User Experience Dashboard.
```
